instrumentation/jetty/jetty-12.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jetty/v12_0/Jetty12ResponseMutator.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.jetty.v12_0;
+
+import io.opentelemetry.javaagent.bootstrap.http.HttpServerResponseMutator;
+import org.eclipse.jetty.server.Response;
+
+public enum Jetty12ResponseMutator implements HttpServerResponseMutator<Response> {
```

Comment:
```suggestion
enum Jetty12ResponseMutator implements HttpServerResponseMutator<Response> {
```

Reply:
this is also used in instrumentation and needs to be public